src/ext/issuance.js

```diff
@@ -33,28 +33,31 @@ const COMMITMENT_URL = "https://raw.githubusercontent.com/privacypass/ec-commitm
 /**
  * Constructs an issue request for sending tokens in Cloudflare-friendly format
  * @param {URL} url URL object for request
+ * @param {Object} details Request details object
  * @return {XMLHttpRequest} XHR info for asynchronous token issuance
  */
-function signReqCF(url) {
+function signReqCF(url, details) {
     const reqUrl = url.href;
-    const manualChallenge = reqUrl.includes("manual_challenge");
-    const captchaResp = reqUrl.includes("g-recaptcha-response");
-    const alreadyProcessed = reqUrl.includes("&captcha-bypass=true");
+    const captchaResp = reqUrl.includes(requestIdentifiers()["query-param"]);
+    const alreadyProcessed = reqUrl.includes(requestIdentifiers()["post-processed"]);
```

Comment:
If suggest to use the [searchParams][2] field instead of performing substring parsing.

```js
const captchaResp = url.searchParams.get(requestIdentifiers()["query-param"]);
// be sure to drop the '&' from the config.js
const alreadyProcessed = url.searchParams.has(requestIdentifiers()["post-processed"]);
// alternatively:
const alreadyProcessed = url.searchParams.get(requestIdentifiers()["post-processed"]) === 'true';
```

 [2]: https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams